job-dsls/jobs/kie/kogito/kogito_tooling_prerelease_branch_UMB_trigger.groovy

```diff
@@ -0,0 +1,108 @@
+/**
+ * Creates pullrequest (PR) jobs for errai
+ */
+import org.kie.jenkins.jobdsl.Constants
+
+//creation of script for log compression
+def errorSh='''#!/bin/bash -e
+cd $WORKSPACE
+touch trace.sh
+chmod 755 trace.sh
+echo "wget  --no-check-certificate  ${BUILD_URL}consoleText" >> trace.sh
+echo "tail -n 1000 consoleText >> error.log" >> trace.sh
+echo "gzip error.log" >> trace.sh'''
+
+def repo = "kogito-tooling"
+def repoBranch = ''
+def ghOrgUnit = "kiegroup"
+def ghAuthTokenId = "kie-ci-token"
+def labelName = "kie-rhel7 && kie-mem16g"
+
+
+// Creation of folders where jobs are stored
+/*
+folder("KIE")
+folder("KIE/${repoBranch}")
+folder("KIE/${repoBranch}/" + Constants.PULL_REQUEST_FOLDER){
+    displayName(Constants.PULL_REQUEST_FOLDER_DISPLAY_NAME)
+}
+def folderPath = ("KIE/${repoBranch}/" + Constants.PULL_REQUEST_FOLDER)
+*/
+
+// jobs for main branch don't use the branch in the name
+//String jobName = "${folderPath}/${repo}-${repoBranch}.pr"
+
+job(jobName) {
+
+    description("Sends UMB message when new prerelease branch is pushed to kogito-tooling")
+
+    logRotator {
+        numToKeep(10)
+    }
+
+    parameters {
+        stringParam("sha1")
+    }
+
+    concurrentBuild()
+
+    properties {
+        ownership {
+            primaryOwnerId("mbiarnes")
+            coOwnerIds("mbiarnes")
+        }
+    }
+
+    jdk(javadk)
+
+    label(labelName)
+
+    // creates script for building error.log.gz
+    steps {
+        shell(errorSh)
+    }
+
+    triggers {
+         genericTrigger {
+                genericVariables {
+                    genericVariable {
+                        key("ref")
+                        value("\$.ref")
+                        regexpFilter("refs\/head\/\[0-9]*\.[0-9]*\.[0-9]*\-prerelease")
+                    }
+                    genericVariable {
+                        key("created")
+                        value("$.created")
+                        regexpFilter("true")
+                    }
+                }
+         }
+
+    }
+
+    wrappers {
+        timeout {
+            elastic(150, 3, 90)
+        }
+        timestamps()
+        colorizeOutput()
+
+        preBuildCleanup()
+    }
+
+    steps {
+        stage ('UMB message'){
+            steps {
+                    if (${ref} && ${created}) {
```

Comment:
not sure I am referencing these vars correctlly

Reply:
I would try if ( "${ref}" && "${created}")